src/acvm_interop.rs

```diff
@@ -1,3 +1,6 @@
+// Allow unwrap because the ProofSystemCompiler interface does not support Result
```

Comment:
We should generalize this comment like
```suggestion
// Allow unwrap because the Backend traits don't support Result.
```